.github/workflows/build.yml

```diff
@@ -256,7 +256,7 @@ jobs:
 
   build-msbuild-i386:
     name: MSBuild (i386)
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
Add a comment to explain why `windows-2019` only?
Can't this job (be made to) work with VS 2022?

Reply:
Ah, msvc jobs use VS 2022 but not its toolset `14.3` yet. Then, fine as is, for the time being.

Reply:
it's similar to the 14.0 toolset problem, but different:
```
CMake Error at CMakeLists.txt:17 (project):
  Generator

    Visual Studio 16 2019
```

Reply:
> `Visual Studio 16 2019`

Sure: from old line 279.
Trying to force `14.2` toolset here seems overkill, hence fine with your "image downgrade" fix.

Reply:
iirc it was unsuccessful by just forcing an older toolset.